sdk/groups_report.go

```diff
@@ -0,0 +1,34 @@
+package sdk
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	apiError "github.com/ONSdigital/dp-identity-api/sdk/errors"
+)
+
+type GroupsReportResponse struct {
```

Comment:
Also, all exported types should have godoc
```suggestion
// GroupsReportResponse is a … that…
type GroupsReportResponse struct {
```